source/agora/test/Script.d

```diff
@@ -229,28 +232,18 @@ unittest
     auto blocks = node_1.getBlocksFrom(2, 1);
     assert(blocks.length == 1);
     sort(txs_0);
-    assert(blocks[0].txs == txs_0);
+    assert(blocks[0].txs.array == txs_0);
```

Comment:
Why ?

Reply:
Otherwise it didn't show the difference.

Reply:
Do you remember why ? It's odd because `txs` is an array already.

Reply:
I just checked and it does work. I'll revert.